app/helpers/title_helper.rb

```diff
@@ -4,6 +4,7 @@ def format_label(count, item)
   end
 
   def format_title(searched, page_title, current_user, item_label, count, organisation_name)
+    organisation_name = sanitise_characters(organisation_name)
     if searched.present?
       actual_title = support_sab_nav?(current_user, organisation_name) ? organisation_name : page_title
       "#{actual_title} (#{count} #{item_label} matching ‘#{searched}’)"
```

Comment:
This format_title method is used in a few places, just using the `sanitise_characters` method here is enough to propagate the changes in all other places.

Reply:
Would using something like this also work? https://apidock.com/rails/ActionView/Helpers/SanitizeHelper/sanitize

Reply:
No, tried that. Unfortunately not in the title

Reply:
Okay, that is frustrating, I had a bit of a dig and I see what you mean in terms of the "&"
Would having something like this instead of writing our own gsub cover all of the cases?
`sanitize(title.tr("&", "＆"))`

Reply:
The gsub is not actually needed, I append the .tr("&", "＆") after it which does all the work. I will remove the gsub part in the `sanitise_characters` method, just included it as extra.

`string.gsub(/[^a-zA-Z0-9\s\-_'&]/, "").tr("'", "’").tr("&", "＆")`
to
`string.tr("'", "’").tr("&", "＆")`